src/zjs_i2c.c

```diff
@@ -110,25 +67,44 @@ static jerry_value_t zjs_i2c_read_base(const jerry_value_t this,
         return zjs_error("zjs_i2c_read_base: buffer creation failed");
     }
 
-    zjs_ipm_message_t send;
-    zjs_ipm_message_t reply;
-
     if (!burst) {
-        send.type = TYPE_I2C_READ;
-    } else {
-        send.type = TYPE_I2C_BURST_READ;
-    }
-
-    send.data.i2c.bus = (uint8_t)bus;
-    send.data.i2c.data = buf->buffer;
-    send.data.i2c.address = (uint16_t)address;
-    send.data.i2c.register_addr = register_addr;
-    send.data.i2c.length = size;
+        if (bus < MAX_I2C_BUS) {
+            // Read has to come after an Open I2C message
```

Comment:
For consistent style, start these next couple comments with lower case. Not critical. :)